app/src/main/java/org/mozilla/focus/fragment/FirstrunFragment.java

```diff
@@ -147,6 +147,13 @@ private void finishFirstrun() {
 
         final UrlInputFragment inputFragment = (UrlInputFragment) fragmentManager.findFragmentByTag(UrlInputFragment.FRAGMENT_TAG);
         if (inputFragment != null) {
+            View view = inputFragment.getView();
+            if (view != null) {
+                View rootView = view.getRootView();
+                if (rootView != null) {
+                    rootView.setImportantForAccessibility(View.IMPORTANT_FOR_ACCESSIBILITY_AUTO);
+                }
```

Comment:
Does this work?

* Shouldn't the root view be the root of the activity and therefore be the same for all fragments? Meaning, doesn't this disable accessibility for all views?
* If Focus was opened from a link then I guess there will be a BrowserFragment and no InputFragment?